content/index.haml

```diff
@@ -17,24 +17,26 @@ title: RVM Ruby Version Manager - Documentation
 
 %ul.square
   %li
-    Install RVM:
+    Install GPG keys:
     = sh_cmd "gpg --keyserver hkp://keys.gnupg.net --recv-keys 409B6B1796C275462A1703113804BB82D39DC0E3"
```

Comment:
maybe switch to `gpg2` without `--keyserver` and add a note to use `gpg` with it?